### DISCUSSION TITLE
running backend on alpine docker image

### DISCUSSION THREAD

**Discussion by PKASHYAP-BEC** [iTwin/itwinjs-core#584] [General] [answered]
Hi , 
I am trying to run our backend in a docker using alpine base image and it is not running. 
it gives "Error loading shared library ld-linux-x86-64.so.2: No such file or directory" error , I tried installing "libc6-compat" but issue is still there, 
then i created a link of actual lib on "/lib64/ld-linux-x86-64.so.2" to "/lib/ld-linux-x86-64.so.2" and it gives "__cmsg_nxthdr: symbol not found" error. 

any help to solve this problem ?

**Comment by jffmarker** [comment, marked as answer]
https://prod.imodeljs.org/v1/learning/supportedplatforms/

We currently support Debian-based distributions, or specifically systems that have GLIBC 2.24 (or greater) and GLIBCXX 3.4.22 (or greater). I believe Alpine is based on MUSL libc, so I suspect you'd have to graft that into the image.
